config/envVarCheck.ts

```diff
@@ -0,0 +1,26 @@
+import { env } from "../src/typeDefinitions/default.types";
+import { loadEnv } from "./config";
+
+validateEnv();
+
+/**
+ * Validate if all the required environment variables are set to a non empty value
+ * else throw an error
+ * ---
+ */
+export function validateEnv() {
+  //pass empty object as env and fromWorkerEnv = false, since this method is should get executed in github actions and not in worker
+  const envLoadedFromProcess: env = loadEnv({}, false);
+  const missingEnvVars = Object.keys(envLoadedFromProcess).filter(
```

Comment:
```suggestion
  const missingEnvVariables = Object.keys(envLoadedFromProcess).filter(
```
NIT

nothing wrong in giving long variable names with clear meaning.
Some people might not understand what `vars` means (non technical people)

Reply:
sure, will update